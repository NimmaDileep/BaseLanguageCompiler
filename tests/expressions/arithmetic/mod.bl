print(5 % 2);
print(0 % 2);
print(-5 % 2);
print(5 % -2);
print(10.5 % 5.3);
print(10 % 2.5);
