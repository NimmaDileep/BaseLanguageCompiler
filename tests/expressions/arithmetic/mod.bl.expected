1
0
-1
1
5.2
0.0